tests/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3"
```

Comment:
Please remove

Reply:
is there any particular way to running tests? I thought docker would be a clean solution for this, isn't it?

Reply:
Only via CI on PR commit

Reply:
If you think it's useful, you can add this file via a different PR while including all relevant local run setup

Reply:
I've removed it and created an npm command for starting a docker instance for use during test runs. Every time clean Redis instance starts. After the test, you can kill the container and you're good to go for the next run.